calico_cni_k8s_test.go

```diff
@@ -83,6 +85,7 @@ var _ = Describe("CalicoCni", func() {
 					panic(err)
 				}
 				containerID, netnspath, session, contVeth, contAddresses, contRoutes, err := CreateContainer(netconf, name)
```

Comment:
clientset.Pods().Create() just creates the API object in the kubernetes API server,
CreateContainer() actually invokes the CNI plugin binary directly. It's NOT the kublet that invokes. This test code invokes it directly passing in similar information to what kubernetes would.